ansible/roles/containerd/tasks/redhat.yaml

```diff
@@ -39,15 +39,13 @@
   delay: 6
   with_items:
     - tar
-    - container-selinux
```

Comment:
why this removed here?
`install container-selinux rpm package` task install container-selinux for all the OS but only if offline_mode_enabled?

Reply:
this task only should install packages for OracleLinux -- in this case, it's just `tar` that it needs. container-selinux should be downloaded for all RHEL like operating systems and that happens on the next task.

at some point, we added this logic and after a few changes we got this very confusing result.